index.js

```diff
@@ -1 +1,87 @@
-// implement your API here
+const express = require('express');
+const db = require('./data/db.js');
+
+const server = express();
+
+server.use(express.json());
+
+server.get('/api/users', (req, res) => {
+    db.find()
+    .then(users => {
+        res.status(200).json(users);
+    })
+    .catch(err => {
+        res.status(500).json({ message: 'The users information could not be retrieved.'});
+    })
+})
+
+server.get('/api/users/:id', (req, res) => {
+    const { id } = req.params;
+    db.findById(id)
+    .then(user => {
+        res.status(200).json(user);
+    })
+    .catch(err => {
+        res.status(404).json({ message: 'The user with the specified ID does not exist.'})
+    })
+})
+
+server.post('/api/users', (req, res) => {
+    if (!req.body.bio || !req.body.name) {
+        res.status(400).json({ message: 'Please provide name and bio for the user.'})
+    }
+    else {
+        db.insert(req.body)
+    .then(edited => {
+        db.findById(edited.id)
+        .then(user => {
+            res.status(201).json(user);
+        })
```

Comment:
make sure to also include a .catch at the end of this block as well in case there are no errors with your POST request but a hiccup happens in this nested findById